static/app/components/onboarding/gettingStartedDoc/utils/useCurrentProjectState.tsx

```diff
@@ -21,12 +21,11 @@ function useCurrentProjectState({
   onboardingPlatforms,
   allPlatforms,
 }: Props) {
-  const params = useParams<{projectId?: string}>();
-  const projectSlug = params.projectId;
   const {projects, initiallyLoaded: projectsLoaded} = useProjects();
   const {selection, isReady} = useLegacyStore(PageFiltersStore);
   const [currentProject, setCurrentProject] = useState<Project | undefined>(undefined);
-
+  const {getParamValue: projectIds} = useUrlParams('project');
+  const projectId = projectIds()?.split('&')[0];
```

Comment:
with `.at()` typescript will tell you that it could be undefined

```suggestion
  const projectId = projectIds()?.split('&').at(0);
```